flopy/utils/get_modflow.py

```diff
@@ -466,7 +468,9 @@ def add_item(key, fname, do_chmod):
                         )
                     ):
                         add_item(key, fname, do_chmod=True)
-        else:  # release 1.0 did not have code.json
+        # executables release 1.0 did not have code.json
```

Comment:
@w-bonelli @langevin-usgs Any thoughts on whether we should include a code.json in MODFLOW 6 or nightly build?

Reply:
Yes, we should probably think about adding code.json to these.  Will probably need to consider if there is anything we'll need to do to get_modflow to manage multiple code.json files in the bin directory.  Maybe a date/time stamp added to the code.json file name?